### PR TITLE
MLPAB-1554 Send certificate provider prompt when LPA ready

### PR DESCRIPTION
**Pull request by hawx**

### CODE REVIEW

File: internal/actor/actoruid/uid.go [hunk 1]
```diff
@@ -8,6 +8,9 @@ import (
 
 const prefix = "urn:opg:poas:makeregister:users:"
 
+// Service is used when the application does an action.
+var Service = UID{value: [16]byte{0, 0, 0, 0, 0, 0, 64}}
```
Comment (hawx) [inline internal/actor/actoruid/uid.go:12]: I didn't want to use the null UUID here, so settled for setting the version to 4. We can always change this later if/when we decide how to represent it. But it shouldn't really matter too much just for getting an LPA as that isn't recorded anywhere.

File: cmd/event-received/main.go [hunk 2]
```diff
@@ -31,6 +31,12 @@ type uidEvent struct {
 	UID string `json:"uid"`
 }
 
+type changeEvent struct {
```
Comment (acsauk) [inline cmd/event-received/main.go:34]: Don't think this is being used anywhere at the moment?

**Review by acsauk — COMMENTED** [review]
One unused type to drop then ready to merge

File: cmd/event-received/main.go (hunk 2 above)
Comment (hawx) [reply cmd/event-received/main.go:34]: ah yeah, left over from when I had the other event

**Review by acsauk — APPROVED** [review]
:shipit:

**Pull request merged**